src/app/authentication/pages/login/login.component.html

```diff
@@ -1,8 +1,8 @@
-<div class="content-wrapper">
-  <div class="row no-gutters">
+<div class="container-fluid content-wrapper">
+  <div class="row no-gutters app-header">
```

Comment:
How about placing custom class name first and framework-related class names after:

```suggestion
  <div class="app-header row no-gutters">
```

It would be easier to see the relation with elements below e.g. `class="app-header__subtitle">`